AppKit/CPSplitView.j

```diff
@@ -1205,6 +1186,32 @@ The sum of the views and the sum of the dividers should be equal to the size of
 
 @end
 
+@implementation CPSplitView (CPTrackingArea)
+
+- (void)updateTrackingAreas
+{
+    var myTrackingAreas = [self trackingAreas];
+    
+    for (var i = 0; i < myTrackingAreas.length; i++)
+        [self removeTrackingArea:myTrackingAreas[i]];
```

Comment:
Indeed, I 100% agree with you. I've already imagined adding such a method. But I would then add a public method, rather than a private. What do you think ?